lib/retry.js

```diff
@@ -19,6 +19,10 @@ import constant from 'lodash/constant';
  * * `interval` - The time to wait between retries, in milliseconds.  The
  *   default is `0`. The interval may also be specified as a function of the
  *   retry count (see example).
+ * * `filter` - Synchronous function that is invoked on erroneous result with the
```

Comment:
`errorFilter`/`errorTest`/`shouldError`/`onError`/`shouldContinueOnError`

I think `continueOperation` is a bit misleading, because it's not something you do when you continue, it's a test/callback to see if you should continue or not.

Cache invalidation and naming things....